qutebrowser/browser/pdfjs.py

```diff
@@ -62,7 +63,7 @@ def generate_pdfjs_page(filename, url):
                         '</body><script>{}</script>'.format(script))
     # WORKAROUND for the fact that PDF.js tries to use the Fetch API even with
     # qute:// URLs.
-    pdfjs_script = '<script src="../build/pdf.js"></script>'
+    pdfjs_script = '<script src="../build/{}"></script>'.format(pdfjs_bin)
```

Comment:
We usually use f-strings for new code:

```suggestion
    pdfjs_script = f'<script src="../build/{html.escape(pdfjs_bin)}"></script>'
```

I also added a [`html.escape` call](https://docs.python.org/3/library/html.html#html.escape) - while strictly speaking we shouldn't have any HTML special chars in it, it's still a footgun for the future if we don't escape the value, and it certainly won't hurt.

Reply:
I have renamed the html local variable to viewhtml as it otherwise shadows the html module identifier